.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
add `.vscode` and `.metals` to `.gitignore` ?

Reply:
yesss I should !